### PR TITLE
ADD: Slide up animation for FloatButton

### PR DESCRIPTION
**Pull request by marcosrdz**
macOS:
https://github.com/BlueWallet/BlueWallet/assets/4793122/2b8f3822-176e-42ce-a5ee-9c36028aab90

iOS:

https://github.com/BlueWallet/BlueWallet/assets/4793122/9790aded-494f-47db-abd1-ed1e63acbc42

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/3fb5wb6zzcij3nrx4l22mhwreu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1168)

**Review by Overtorment — APPROVED** [review]

File: components/FloatButtons.tsx [hunk 1]
```diff
@@ -39,6 +39,18 @@ export const FContainer = forwardRef<View, FContainerProps>((props, ref) => {
...
+
+  useEffect(() => {
+    slideAnimation.setValue(height);
+    Animated.spring(slideAnimation, {
+      toValue: 0,
+      useNativeDriver: true,
+      speed: 100,
+      bounciness: 3,
+    }).start();
+  }, [height, slideAnimation]);
```
Comment (limpbrains) [inline components/FloatButtons.tsx:53]: What about desktop app, will animation run every time window has been resized?

**Review by limpbrains — COMMENTED** [review]
I didn't do animation for Float buttons at the first place, because I thought it would be annoying. 
Can you please share a video of how it looks like.

File: components/FloatButtons.tsx (hunk 1 above)
Comment (marcosrdz) [reply components/FloatButtons.tsx:53]: @limpbrains  video attached as description

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**